content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -169,6 +171,29 @@ CREATE TABLE "Example" (
 CREATE INDEX "Example_val_idx" ON "Example" USING HASH (val);
 ```
 
+### Configuring if indexes are clustered or non-clustered with `clustered` (SQL Server)
+
+The `clustered` argument is available to configure (non)clustered indexes in SQL Server in version `3.13.0` and later. It can be used on `@id`, `@@id`, `@unique`, `@@unique` and `@@index` attributes.
+
+As an example, the following model configures the `@id` to be non-clustered (instead of the clustered default):
+
+```prisma file=schema.prisma
+model Example {
+  id    Int @id(clustered: false)
+  value Int
+}
+```
+
+This translates to the following SQL commands:
+
+```sql
+CREATE TABLE [Example] (
+  id INT NOT NULL,
+  value INT,
+  CONSTRAINT [Example_pkey] PRIMARY KEY NONCLUSTERED (id)
+)
+```
```

Comment:
actual change 3

Reply:
Defaults:
- `id`: `clustered: true`
- `unique`: `clustered: false`
- `index`: `clustered: false`

A table can have at most one clustered index.

Reply:
Can you take over adding a table of the defaults here @keerlu? Thanks.

Reply:
Added this in